ice.config

```diff
@@ -1,4 +1,4 @@
-omero.host=omero
```

Comment:
This fails docker since the name of the server container is "omero":
```
Exception in thread "main" omero.gateway.exception.DSOutOfServiceException: java.net.ConnectException: Connection refused
1871	at omero.gateway.Gateway.connect(Gateway.java:304)
1872	at com.example.SimpleConnection.connect(SimpleConnection.java:73)
1873	at com.example.SimpleConnection.main(SimpleConnection.java:121)
1874Caused by: Ice.ConnectionRefusedException
1875    error = 0
```

Reply:
That's why, it was set to ``omero``. Rolling back that commit